modules/cluster-logging-elasticsearch-retention.adoc

```diff
@@ -16,8 +16,8 @@ Elasticsearch rolls over an index, moving the current index and creating a new
 index, when an index matches any of the following conditions:
 
 * The index is older than than the `rollover.maxAge` value in the Elasticsearch CR.
-* The index size is greater than 40gb x number of primary shards.
-* The index doc count is greater than 40960 x number of primary shards.
+* The index size is greater than 40gb x the number of primary shards.
+* The index doc count is greater than 40960k x the number of primary shards.
```

Comment:
Per IBM SG seems the style should be:

40 GB × the number [...]
40960 KB × the number [...]

Uppercase abbrev with a space and with (×) being an actual multiplication sign.